ethers-solc/src/lib.rs

```diff
@@ -481,6 +481,49 @@ impl<Artifacts: ArtifactOutput> Project<Artifacts> {
         }
         Ok(())
     }
+
+    /// Flatten all file imports into a single string
+    pub fn flatten(&self, target: &PathBuf) -> Result<String> {
```

Comment:
since the graph only requires the `ProjectPathsConfig`, I'd rather move this to `ProjectPathsConfig::flatten`